req.go

```diff
@@ -783,10 +783,101 @@ type ReqFilter struct {
 	AttributeRegexp    map[string]*regexp.Regexp
 }
 
+// createFilterFromCmdLine reads the filter regular expressions from the command line arguments and
+// compiles them into a filter structure ready to use
+// @llr REQ-TRAQ-SWL-19, REQ-TRAQ-SWL-73
+func createFilterFromCmdLine(idFilter, titleFilter, bodyFilter string, attributeFilter []string) (ReqFilter, error) {
```

Comment:
`FromCmdLine` doesn't seem very accurate anymore. Maybe just remove it or use `FromParts`.